packages/gui/src/components/pool/PoolOverview.tsx

```diff
@@ -132,6 +133,8 @@ export default function PoolOverview() {
   const hasNFTs =
     (!!nfts && !!nfts?.length) || !!external?.length || unconfirmed.length;
 
+  const passedNFTLimit = !(hasNFTs && nfts?.length >= 50);
```

Comment:
I'm confused by this, and I'm not sure if it's just a variable naming issue or if the logic is wrong.

Is the intent of the name `exceededNFTLimit` as in we've exceeded the NFT limit? Or is it `passed` as in we passed the test that we're below the NFT limit?

From the code below, I suspect it's the former `exceededNFTLimit`, in which case shouldn't the assignment be:

`const exceededNFTLimit = nfts?.length >= 50;`

Reply:
Sorry it should be exceededNFTLimit, thanks!